src/components/elements/table/GenericTable.tsx

```diff
@@ -176,10 +188,48 @@ export const renderHeaderCellContents = <T extends { id: string }>(
   );
 };
 
+export const renderLinkedCellContents = <T extends { id: string }>(
+  rowLink: To,
+  row: T,
+  render: ColumnDef<T>['render'],
+  index: number,
+  rowLinkState: LocationState | undefined = undefined
+) => {
+  return (
+    <RouterLink
+      to={rowLink}
+      state={rowLinkState}
+      plain
+      sx={{
+        height: '100%',
+        verticalAlign: 'middle',
+        display: 'block',
+        '&.Mui-focusVisible': {
+          outlineOffset: '-2px',
+        },
```

Comment:
I'm not seeing the outline visually when testing. This fixes it for me:
```
          outline: 'auto',
          outlineOffset: '-5px',
```
(5px so it doesn't get unevenly cut off, but maybe that's preferable to having it more inside the cell - could go either way

Reply:
Thanks! I think -5px is good. 

`outline: 'auto'` for me makes the outline look black, so I also added an `outlineColor` that matches what we apply to buttons in our `theme.ts`. Interesting that this is treated differently in Chrome than Safari.